modules/standard/Barriers.chpl

```diff
@@ -319,63 +319,96 @@ module Barriers {
   /* A task barrier implemented using sync and single variables. Can be used
      as a simple barrier or as a split-phase barrier.
    */
-  pragma "no doc" class sBarrier: BarrierBaseType {
+ pragma "no doc" class sBarrier: BarrierBaseType {
+    /* If true the barrier can be used multiple times.  When using this as a
+       split-phase barrier this causes :proc:`wait` to block until all tasks
+       have reached the wait */
+    param reusable = true;
+
+    pragma "no doc"
```

Comment:
Unless you feel strongly, I'll probably just leave this as-is since most of these fields already had `no doc` and the ones in the aBarrier class do as well.

Reply:
👍